wp-content/mu-plugins/pub/locale-switcher/package.json

```diff
@@ -14,6 +14,12 @@
 		"packages-update": "wp-scripts packages-update"
 	},
 	"devDependencies": {
-		"@wordpress/scripts": "12.1.1"
+		"@wordpress/scripts": "^19.2.2"
```

Comment:
The build for this package was failing and upgrading this package to match the others in this repo fixed it. 

From what I can tell it seems updating the translation strings downgraded it 🤔 
https://github.com/WordPress/Learn/blame/trunk/wp-content/mu-plugins/pub/locale-switcher/package.json

Reply:
That's so strange, I can't find anything in the workflow that would do that. I wonder if the same thing will happen when you merge this PR.